app/controllers/ops_controller/diagnostics.rb

```diff
@@ -366,17 +366,6 @@ def log_protocol_changed
     render :json => log_depot_json
   end
 
-  def db_gc_collection
-    begin
-      MiqSchedule.run_adhoc_db_gc(:userid => session[:userid])
```

Comment:
We should probably create a backend PR that drops the above routine (since it doesn't seem to be used).

Reply:
https://github.com/ManageIQ/manageiq/pull/20877